cloud/shared/bin/lib/config_loader.py

```diff
@@ -12,16 +12,12 @@
   Provides getters to return values from the config. 
 """
 class ConfigLoader:
-    TF_VARS_FILENAME = "setup.auto.tfvars"
-    BACKEND_CONFIG_FILENAME = "backend_vars"
-    
-    @property
+
     def tfvars_filename(self):
-        return self.TF_VARS_FILENAME
+        return os.environ.get('TF_VAR_FILENAME', None)
```

Comment:
we probably would want to fail here it can't be null?

Reply:
done